### ISSUE TITLE
[fr] Extra tagging for "V ind pres 1 s" special forms (in inverted questions)

### ISSUE THREAD

**Issue by udomai** [languagetool-org/languagetool#5739]
Idea by @jaumeortola:

Give the forms "pensé" and "pensè" a special tag for synthesizing inverted questions:

Je pense quoi ? → Que **pensé**-je ? / Que **pensè**-je ?

This is relevant only for the 1st person sg. indicative present. Does it make sense to add the extra tag to verbs of all groups, even those that don't change ?

Que **resous**-je ?

Also, should be maybe just propose the more popular spelling "pensé" until such a time when we can actively monitor uniform usage of the _réforme de l'orthographe_?

@Rebecca-auque @remymartinDev, do you have any input/feedback on this?

**Comment by remymartinDev** [comment]
I don't think that this extra-tag should be extended to 2nd/3rd group verbs by default , except for "pouvoir" {"je peux"->"puis-je"} (but maybe there are others that I don't think of right away)

From what I know and what I've read (but I only read contemporary authors and almost no "old" literature) , the form I have encountered the most is the "é" form (more than the "è" form)

Anyway, this tag will be very helpful

**Comment by Rebecca-auque** [comment]
Same as  @remymartinDev, I'd put it only for 1st group verbs but I think it would be good to create it for both spellings (old/new : -é/-è) just to be fair as it would be right to suggest only one over the other, just in case the user wants to use the new spelling for an unknown reason.

**Comment by jaumeortola** [comment]
Thinking about it, the tagging alone is not going to solve all the needs, and it can create some problems in existing rules. 
We will need a filter anyway,  so let's keep it simple and not add more tagging. I will adjust the existing filter to do what is needed.

**Comment by udomai** [comment]
Okay! This will be immediately useful for @remymartinDev's `INTERROGATIVE_DIRECTE`. Thank you very much in advance 🤓
